devicemaps.json

```diff
@@ -1638,11 +1638,11 @@
             "type": "LAN",
             "speed": 10000,
             "name": "xe-0-1-0",
-            "description": "Port 8 labeled on the device",
+            "description": "Port 1/0 labeled on the device",
```

Comment:
For LAN ports with non `0-0-X`, how are we going to predictably determine an ip? When these had numbers (like this was `port 8`), then we would just do `192.168.8.1`

Reply:
Do we need this resolved as part of this PR? These sorts of multi-slot devices are going to have overlapping port numbers no matter what.